migreringer/src/main/resources/db/postgres/defaultDS/1.0/V1.0_055_oppretter_crontask_for_oppfriskning_av_registerinntekt.sql

```diff
@@ -0,0 +1,3 @@
+insert into prosess_task (id, task_type, prioritet, status, task_gruppe, task_sekvens, partition_key)
+select nextval('SEQ_PROSESS_TASK'), 'batch.oppfriskInntektPåVent', 1, 'KLAR', nextval('SEQ_PROSESS_TASK_GRUPPE'), 1,'05'
+    where not exists (select 1 from prosess_task where task_type = 'batch.oppfriskInntektPåVent'  and status IN ('KLAR', 'FEILET'));
```

Comment:
Fleire av feltene her settes som default ved opprettelse. Eg trur egentlig dette burde vere nok
```suggestion
insert into prosess_task (id, task_type, task_gruppe)
select nextval('SEQ_PROSESS_TASK'), 'batch.oppfriskInntektPåVent', nextval('SEQ_PROSESS_TASK_GRUPPE')
where not exists (select 1 from prosess_task where task_type = 'batch.oppfriskInntektPåVent'  and status IN ('KLAR', 'FEILET'));
```